recipes/melodies-monet/meta.yaml

```diff
@@ -0,0 +1,70 @@
+{% set name = "melodies-monet" %}
+{% set version = "0.1.dev1" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/NOAA-CSL/MELODIES-MONET/archive/v{{ version }}.tar.gz
+  sha256: 11348c50e447bb54e6a9f8c8ead71c6ee748d97f58b532efbb0d098f9b882ecd
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+  entry_points:
+    - "melodies-monet = melodies_monet._cli:cli"
+
+requirements:
+  host:
+    - python >=3.7
+    - pip
+    - setuptools
+  run:
+    - python >=3.7
+    #
```

Comment:
Let's save a few lines. The comment below is good enough to separate the sections.
```suggestion
```